### PR TITLE
CTRL Click on PDA's will now pull pens

### PR DESCRIPTION
**Pull request by Fruerlund**
In the past you could retrieve your ID by ALT clicking on PDAs. Today you can retrieve your pen by CTRL clicking on your PDA... Hopefully

🆑Fruerlund
add: You can now take out pens by CTRL Clicking PDAs
/🆑

### CODE REVIEW

**Comment by tigercat2000** [comment]
This will mean that you both rip a pen out of a PDA and start to pull it if it's on the floor.

File: code/modules/pda/PDA.dm [hunk 1]
```diff
@@ -338,31 +346,32 @@ var/global/list/obj/item/device/pda/PDAs = list()
...
+
+/obj/item/device/pda/proc/remove_pen()
 
 	if(issilicon(usr))
 		return
 
 	if( can_use(usr) )
 		var/obj/item/weapon/pen/O = locate() in src
 		if(O)
+			to_chat(usr, "<span class='notice'>You remove \the [O] from \the [src].</span>")
```
Comment (Fox-McCloud) [inline code/modules/pda/PDA.dm]: if you're going to do this, don't use `usr`. It's gross.

You should be using `user` and properly passing the argument.

Comment (Fruerlund) [inline code/modules/pda/PDA.dm]: @Fox-McCloud could you explain what you mean by properly passing the argument?

**Comment by Fruerlund** [comment]
@tigercat2000 not sure what you mean. This is pretty much the same code from when you pull out IDs. I cannot seem to replicate the issue you are refering too

**Comment by Ralta** [comment]
Yeeeeeeees. I use pens a lot in medbay to name pill bottles and write tags and such; this would save me ALL the clicks. 
+1

File: code/modules/pda/PDA.dm (hunk 1 above)
Comment (Crazylemon64) [inline code/modules/pda/PDA.dm]: In `CtrlClick`, pass `remove_pen` the argument `user`. Then, give this proc a single argument - `mob/user`, and replace all uses of `usr` with `user`. Finally, you can put `usr` as the first argument to `remove_pen` in `verb_remove_pen`

**Comment by taukausanake** [comment]
@Fruerlund I think he means if you Ctrl-Click a PDA on the floor rather than in your inventory. Unless he thinks you'll pull the pen that you didn't Ctrl-Click, that's weird.
I think I rather have if a verb instead of a key+click but no one likes verbs for some reason. Also weird

**Comment by Crazylemon64** [comment]
choo choo full speed ahead

**Pull request merged**